homeassistant/components/airvisual/strings.json

```diff
@@ -18,6 +18,13 @@
           "password": "[%key:common::config_flow::data::password%]"
         }
       },
+      "reauth_confirm": {
```

Comment:
When this goes out in a hot fix, we need to include new translations.

Reply:
I ran `translations develop` – what else do I need to do?

Reply:
This doesn't _have_ to go out in a patch release, so if the easiest thing to do is to remove the milestone, I'm fine with that.

Reply:
Oh with translations develop it will work.